config/localization/en/app.json

```diff
@@ -1326,6 +1326,9 @@
       "NEXT_IN": "Next in {TIME}",
       "NOTIONAL_VALUE": "Notional Value",
       "OFFSETTING": "Offsetting",
+      "OHLC": "OHLC",
+      "OHLC_VIA_MID_PRICE": "OHLC (via mid-price)",
+      "OHLC_TOOLTIP": "Enable or disable OHLC data via mid-price. Disabling this will only display OHLC data from the orderbook.",
```

Comment:
@aforaleka mentioned [the tooltip should maybe be under the tooltip category](https://dydx-team.slack.com/archives/C054WQ613D4/p1721835397546649?thread_ts=1721834388.956619&cid=C054WQ613D4)